eng/pipelines/notifications.yml

```diff
@@ -76,23 +77,13 @@ stages:
             --organization $(Organization) `
             --project $(Project) `
             --path-prefix "\$(PathPrefix)" `
-            --dev-ops-token-var DEVOPS_TOKEN `
-            --aad-app-id-var APP_ID `
-            --aad-app-secret-var APP_SECRET `
-            --aad-tenant-var AAD_TENANT `
-            --kusto-url-var KUSTO_URL `
-            --kusto-database-var KUSTO_DB `
-            --kusto-table-var KUSTO_TABLE `
+            --dev-ops-token-var $(azure-sdk-notification-tools-pat) `
+            --aad-app-id-var $(opensource-aad-app-id) `
```

Comment:
Looks like you still called these command line parameters with "-var" but you are not passing a variable and instead are passing the value. We should probably follow-up with an update to fix that. We should decide if they should be variables or values.

Reply:
I thought it means the value is from variable in pipeline. Make sense to me :)

Reply:
Put it in issue: https://github.com/Azure/azure-sdk-tools/issues/2745